### ISSUE TITLE
config.php ... wich one ?

### ISSUE THREAD

**Issue by cr9christi9n** [BesrourMS/clean-blog#11]
Hi,
Many thanks for your job.
I try to set up your theme in my Nextcloud and I wonder (sorry for this stupid question ...) wich config.php do you speak in your README.md file ?

**Comment by cr9christi9n** [comment]
... I've got this error sign also ....
![image](https://user-images.githubusercontent.com/61017785/80234986-52590b00-8659-11ea-8a61-71c4336f2b5e.png)

**Comment by BesrourMS** [comment]
Hi @cr9christi9n 
For Pico CMS under version 2 we have an config php but if you use Pico CMS >= 2.0 add
 ```theme: clean-blog``` 
to config/config.yml

**Comment by cr9christi9n** [comment]
Hi BesrourMS,
Sorry but I don't understand where is that file : config.yml ?

**Comment by BesrourMS** [comment]
Hi @cr9christi9n,
The file does not exist, create it. For more info see config/config.yml.template

**Comment by cr9christi9n** [comment]
Hi @BesrourMS 
OK I edited config.yml, the theme is now set up.
I also set up the plugin 'pagination' and uploaded index.md and post.md to my content folder.
However after many tries, I can't access to index page (or anyone else) of my test site, I've got an internal error of my server.
(I first tried on this server with default pico theme and template and everything were going on well).
!??
Many thanks any way for your theme and help.
